tests/Report/junit.xsd

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
for the current state of PR - how could one know which specification are we following ?